api/MouseEvent.json

```diff
@@ -50,6 +50,168 @@
           "deprecated": false
         }
       },
+      "MouseEvent": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/MouseEvent/MouseEvent",
+          "description": "<code>MouseEvent()</code> constructor",
+          "support": {
+            "webview_android": {
+              "version_added": "47"
+            },
+            "chrome": {
+              "version_added": "47"
```

Comment:
the MDN page says the constructor got support in Chrome in v45

Reply:
Yes, the root page indicated v45. However, the [constructor page](https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent/MouseEvent#Browser_compatibility) said otherwise. When in doubt, I use the later version.